From a96d83107381564a44a6964003d267dd80df8a1c Mon Sep 17 00:00:00 2001
From: Hood Chatham <roberthoodchatham@gmail.com>
Date: Sun, 17 Jul 2022 14:40:39 +0100
Subject: [PATCH 1/5] Public pymain_run_python

---
 Modules/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/main.c b/Modules/main.c
index 7edfeb3365..f5f30c7043 100644
--- a/Modules/main.c
+++ b/Modules/main.c
@@ -546,7 +546,7 @@ pymain_repl(PyConfig *config, int *exitcode)
 }
 
 
-static void
+void
 pymain_run_python(int *exitcode)
 {
     PyObject *main_importer_path = NULL;
-- 
2.25.1

