From 880f1c4c2528fdf58a3382b7b491d054c33fa9f4 Mon Sep 17 00:00:00 2001
From: Hood Chatham <roberthoodchatham@gmail.com>
Date: Thu, 11 Jul 2024 13:53:35 +0200
Subject: [PATCH 2/2] Fix PyEM trampoline calls

---
 Include/internal/pycore_object.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Include/internal/pycore_object.h b/Include/internal/pycore_object.h
index 7602248f956..59ca32b4751 100644
--- a/Include/internal/pycore_object.h
+++ b/Include/internal/pycore_object.h
@@ -731,11 +731,6 @@ PyAPI_FUNC(PyObject*) _PyObject_GetState(PyObject *);
  * trampoline mitigates common occurrences of bad fpcasts on Emscripten.
  */
 #if defined(__EMSCRIPTEN__) && defined(PY_CALL_TRAMPOLINE)
-#define _PyCFunction_TrampolineCall(meth, self, args) \
-    _PyCFunctionWithKeywords_TrampolineCall( \
-        (*(PyCFunctionWithKeywords)(void(*)(void))(meth)), (self), (args), NULL)
-extern PyObject* _PyCFunctionWithKeywords_TrampolineCall(
-    PyCFunctionWithKeywords meth, PyObject *, PyObject *, PyObject *);
 #else
 #define _PyCFunction_TrampolineCall(meth, self, args) \
     (meth)((self), (args))
-- 
2.34.1

