From 3b97366cb5f3bce29e57e13422ea8053bb98e8cd Mon Sep 17 00:00:00 2001
From: Hood Chatham <roberthoodchatham@gmail.com>
Date: Fri, 9 Jun 2023 14:53:40 -0700
Subject: [PATCH 5/5] Don't build all for libinstall

---
 Makefile.pre.in | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index e2adc3cb49..d4ac2dfc28 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -878,14 +878,6 @@ $(DLLLIBRARY) libpython$(LDVERSION).dll.a: $(LIBRARY_OBJS)
 # wasm assets directory is relative to current build dir, e.g. "./usr/local".
 # --preload-file turns a relative asset path into an absolute path.
 
-.PHONY: wasm_stdlib
-wasm_stdlib: $(WASM_STDLIB)
-$(WASM_STDLIB): $(srcdir)/Lib/*.py $(srcdir)/Lib/*/*.py \
-	    $(srcdir)/Tools/wasm/wasm_assets.py \
-	    Makefile pybuilddir.txt Modules/Setup.local
-	$(PYTHON_FOR_BUILD) $(srcdir)/Tools/wasm/wasm_assets.py \
-	    --buildroot . --prefix $(prefix)
-
 python.html: $(srcdir)/Tools/wasm/python.html python.worker.js
 	@cp $(srcdir)/Tools/wasm/python.html $@
 
@@ -2229,7 +2221,7 @@ COMPILEALL_OPTS=-j0
 TEST_MODULES=@TEST_MODULES@
 
 .PHONY: libinstall
-libinstall:	all $(srcdir)/Modules/xxmodule.c
+libinstall: $(srcdir)/Modules/xxmodule.c
 	@for i in $(SCRIPTDIR) $(LIBDEST); \
 	do \
 		if test ! -d $(DESTDIR)$$i; then \
-- 
2.25.1

