--- a/Makefile.pre.in	2025-04-20 03:09:47
+++ b/Makefile.pre.in	2025-05-03 04:11:41
@@ -684,12 +684,7 @@
 
 LIBHACL_HMAC_OBJS= \
 		Modules/_hacl/Hacl_HMAC.o \
-		Modules/_hacl/Hacl_Streaming_HMAC.o \
-		$(LIBHACL_MD5_OBJS) \
-		$(LIBHACL_SHA1_OBJS) \
-		$(LIBHACL_SHA2_OBJS) \
-		$(LIBHACL_SHA3_OBJS) \
-		$(LIBHACL_BLAKE2_OBJS)
+		Modules/_hacl/Hacl_Streaming_HMAC.o
 
 LIBHACL_HEADERS= \
 		Modules/_hacl/include/krml/FStar_UInt128_Verified.h \
